services/QuillLMS/app/views/application/_footer.html.erb

```diff
@@ -9,8 +9,7 @@
 				<div class="top-half">
 					<div class="logo-and-text">
 						<a href="/"><img src="https://assets.quill.org/images/logos/quill-gray-logo-footer.svg"></img></a>
-
-						<a class="footer-copy" href="http://community.quill.org/">Quill is a free literacy tool powered by an open source, non-profit community of educators and developers.</a>
```

Comment:
Truly disconcerting to me that we've been linking to it in the footer this whole time.